proto/cmp/types/v3/currency.proto

```
@@ -0,0 +1,1539 @@
+  // - Entities: AFGHANISTAN
+  ISO_CURRENCY_AFA = 4;
+
+  // - [HISTORICAL: 2002-03]
+  // - Currency: Markka [FIM, 246]
+  // - Entities: ÅLAND ISLANDS, FINLAND
+  ISO_CURRENCY_FIM = 246;
+
+  // - [HISTORICAL: 1989-12]
+  // - Currency: Old Lek [ALK, 008]
+  // - Entities: ALBANIA
+  ISO_CURRENCY_ALK = 8;
```

buf (1.47.2)

enum cmp.types.v3.IsoCurrency: values ISO_CURRENCY_ALL and ISO_CURRENCY_ALK both have the same numeric value 8; use allow_alias option if intentional

(COMPILE)